src/index.js

```diff
@@ -1,51 +1,135 @@
 
 import './style.css';
 const divContainer = document.querySelector(".books-container");
+const inputDiv = document.querySelector("#title");
+const inputSubmit = document.querySelector("#submit");
 
 
+let todoList = [];
+let isEdit = false
+let editId = null;
 
-const todoList = [
-    {
-      description: 'Go shopping for lunch',
-      completed: false,
-      index: 0,
-    },
-    {
-      description: 'Take kids to school',
-      completed: false,
-      index: 1,
-    },
-    {
-      description: 'Work on coding challenges',
-      completed: false,
-      index: 2,
-    },
-    {
-        description: 'Go to the gym',
-        completed: false,
-        index: 3,
-      },
-    {
-      description: 'Attend workshop',
-      completed: false,
-      index: 4,
-    },
-  ];
-
-  const displayTodo = ({ description, index }) => {
-    const divElement = document.createElement('div');
-    divElement.className = 'first-item';
-    divElement.innerHTML = `
+console.log(editId)
+
+
+
+  const setLocalStorage = (todoList)=>{
+    localStorage.setItem('formInputs', JSON.stringify(todoList));
+  }
+
+  const getLocalStorage = ()=>{
+    if (localStorage.getItem('formInputs') !== null) {
+      todoList = JSON.parse(localStorage.getItem('formInputs'));
+      } else {
+      todoList = [];
+
+      }
+      return todoList;
+  }
+
+  const displayTodo = ({ description, index}) => {
+    
+      const divElement = document.createElement('div');
+
+      divElement.className = 'first-item';
+      divElement.innerHTML = `
       <div class="item-details">
       <input type="checkbox" id="" name="" value=""> <h4 class="item-desription">${description}</h4>
       </div>
-      <i class="fa-solid fa-trash-can" id"${index}"></i>
-      `;
+      
+      <button class="edit">edit</button>
+      <button type="button" class="delete" id="${index}">del</button>
+            `;
   
     return divElement;
+    
+    
+  
   };
+
+  inputSubmit.addEventListener('click', (e) => {
+    e.preventDefault();
+      if(inputDiv.value.trim()!== ''){
+        
+          const list = {
+            description: inputDiv.value,
+            completed: false,
+            index: todoList.length +1, 
+        };
+        console.log(list);
+        todoList.push(list);
+        setLocalStorage(todoList);
+        todoList = getLocalStorage();
+        divContainer.innerHTML='';
+        console.log(todoList);
+        todoList.forEach((item)=>{
+          divContainer.append(displayTodo(item));
+          
+        })
+        isEdit = false;
+        
+    
+        }; 
+        inputDiv.value = ``;
+        removeEvents();
+        editEvents();
   
-  todoList.forEach((todo) => {
-    divContainer.append(displayTodo(todo));
   });
   
+
+  const reassignIndex = (todoList) => {
+    todoList.forEach((item, i) => {
+      item.index = i + 1;
+    });
+  }
+  
+  const deleteTask = (id) => {
+    let todoList = getLocalStorage();
```

Comment:
I suggest you fix this method of getting things from the local storage so that once I refresh the browser, I can get back the items already inside the local storage. Currently, the age refreshes and all items disappear. When trying to add a  new item, the local storage resets. See the video attached 

https://user-images.githubusercontent.com/48546992/199698427-01e61e82-0147-468a-8427-f9af8c1a2b7d.mov

Reply:
Fixed.